app/components/error_list/error_list.js

```diff
@@ -0,0 +1,98 @@
+// Copyright (c) 2017 Mattermost, Inc. All Rights Reserved.
+// See License.txt for license information.
+
+import React, {PropTypes, PureComponent} from 'react';
+import {
+    Dimensions,
+    StyleSheet,
+    View,
+    TouchableOpacity
+} from 'react-native';
+import Icon from 'react-native-vector-icons/FontAwesome';
+
+import FormattedText from 'app/components/formatted_text';
+
+import GeneralError from './general_error';
+
+const {width: deviceWidth} = Dimensions.get('window');
+
+const style = StyleSheet.create({
+    closeButton: {
+        height: 20,
+        width: 20,
+        borderRadius: 10,
+        marginBottom: 5,
+        borderColor: '#fff',
+        borderWidth: 1,
+        alignItems: 'center',
+        justifyContent: 'center'
+    },
+    closeButtonContainer: {
+        alignItems: 'center',
+        justifyContent: 'center',
+        marginTop: 5
+    },
+    closeButtonText: {
+        color: '#fff'
+    },
+    container: {
+        paddingTop: 15,
+        paddingBottom: 15,
+        alignItems: 'center',
+        justifyContent: 'center',
+        minHeight: 75
+    },
+    wrapper: {
+        position: 'absolute',
+        top: 0,
+        left: 0,
+        width: deviceWidth,
+        overflow: 'hidden',
+        backgroundColor: 'rgba(255, 116, 92, 1)',
+        zIndex: 99999
+    }
+});
+
+export default class ErrorList extends PureComponent {
+    static propTypes = {
+        actions: PropTypes.shape({
+            dismissError: PropTypes.func.isRequired
+        }).isRequired,
+        errors: PropTypes.array.isRequired
+    }
+
+    render() {
+        return (
+            <View style={[style.wrapper, (!this.props.errors.length && {height: 0})]}>
```

Comment:
I'm actually not sure what would be the purpose of showing the user most errors. Connection errors make sense and maybe general "an error has occurred" messages, but apart from that, I don't think we should show them by default. On pre-release, they're only displayed because we have developer mode enabled in the System Console.

What's more important to me is having them logged to the server or something so that an admin can easily view them when contacting us for support